src/groups/dto/responses/group-with-member-count.dto.ts

```diff
@@ -11,9 +11,20 @@ export class GroupWithMemberCountDto extends GroupResponseDto {
   @Expose()
   memberCount: number;
 
+  @ApiProperty({
+    description: '그룹 이미지 사진',
+    example: 'https://amazonaws.com/group_banner/1/default.png',
+  })
   @Expose()
   groupImageUrl: string;
 
+  @ApiProperty({
+    description: '그룹 배너 이미지 사진',
+    example: 'https://amazonaws.com/group_banner/1/default.png',
+  })
+  @Expose()
+  groupBannerUrl: string | null;
```

Comment:
nullable도 문서도 포함하면 좋을거같아요

```suggestion
  @ApiProperty({
    description: '그룹 배너 이미지 사진',
    example: 'https://amazonaws.com/group_banner/1/default.png',
    nullable: true,
  })
  @Expose()
  groupBannerUrl: string | null;
```